fawltydeps/settings.py

```diff
@@ -93,6 +93,27 @@ def parse_path_or_stdin(arg: str) -> PathOrSpecial:
     return Path(arg)
 
 
+default_ignore_unused = set(
+    [  # Development tools not meant to be imported
+        "black",
+        "codespell",
+        "colorama",
+        "coverage",
+        "fawltydeps",
+        "flake8",
+        "hypothesis",
+        "isort",
+        "mypy",
+        "nox",
+        "pylint",
+        "pyright",
+        "pytest",
+        "wheel",
+        "tox",
+    ]
+)
```

Comment:
I think I prefer this constant with UPPER_CASE spelling. Also, suggest to use the `{ ... }` literal set syntax instead of `set([ ... ])`:

```suggestion
DEFAULT_IGNORE_UNUSED = {
    # Development tools not meant to be imported
    "black",
    "codespell",
    "colorama",
    "coverage",
    "fawltydeps",
    "flake8",
    "hypothesis",
    "isort",
    "mypy",
    "nox",
    "pylint",
    "pyright",
    "pytest",
    "wheel",
    "tox",
}
```

Reply:
:+1: 
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/b6afa98689e4720130fad434d0ce13e2d2b44a26.